platform/app/public/config/local_orthanc.js

```diff
@@ -24,6 +24,7 @@ window.config = {
         wadoUriRoot: 'http://localhost/dicom-web',
         qidoRoot: 'http://localhost/dicom-web',
         wadoRoot: 'http://localhost/dicom-web',
+        acceptHeader: ['multipart/related; type=application/octet-stream; transfer-syntax=*'],
```

Comment:
I think this should be encoded as the default, with appropriate quotes added, rather than embedding here everywhere.

Reply:
Tweak generateAcceptHeaders to have a acceptHeader.push('transfer-syntax=*')
OR set the requestTransferSyntax to be * by default, and if it is set to false then remove